packages/esm-outpatient-app/src/active-visits/active-visits-table.resource.ts

```diff
@@ -78,23 +111,42 @@ export function useVisitQueueEntries(): UseVisitQueueEntries {
     openmrsFetch,
   );
 
-  const mapQueueEntryProperties = (queueEntry: QueueEntry): MappedQueueEntry => ({
-    id: queueEntry.uuid,
-    name: queueEntry.display,
-    patientUuid: queueEntry.patient.uuid,
+  const mapEncounterProperties = (encounter: Encounter): MappedEncounter => ({
+    diagnoses: encounter.diagnoses,
+    encounterDatetime: encounter.encounterDatetime,
+    encounterType: encounter.encounterType.display,
+    obs: encounter.obs,
+    provider: encounter.encounterProviders[0]?.provider?.person?.display,
+    uuid: encounter.uuid,
+    voided: encounter.voided,
+  });
+
+  const mapVisitQueueEntryProperties = (visitQueueEntry: VisitQueueEntry): MappedVisitQueueEntry => ({
+    id: visitQueueEntry.queueEntry.uuid,
+    encounters: visitQueueEntry.visit.encounters.map(mapEncounterProperties),
+    name: visitQueueEntry.queueEntry.display,
+    patientUuid: visitQueueEntry.queueEntry.patient.uuid,
     // Map `Urgent` to `Priority` because it's easier to distinguish between tags named
     // `Priority` and `Not Urgent` rather than `Urgent` vs `Not Urgent`
-    priority: queueEntry.priority.display === 'Urgent' ? 'Priority' : queueEntry.priority.display,
-    priorityComment: queueEntry.priorityComment,
-    service: queueEntry.queue.service.display,
-    status: queueEntry.status.display,
-    waitTime: queueEntry.startedAt ? `${Math.abs(dayjs().diff(dayjs(queueEntry.startedAt), 'minutes'))}` : '--',
+    priority:
+      visitQueueEntry.queueEntry.priority.display === 'Urgent'
+        ? 'Priority'
+        : visitQueueEntry.queueEntry.priority.display,
+    priorityComment: visitQueueEntry.queueEntry.priorityComment,
+    service: visitQueueEntry.queueEntry.queue.service.display,
+    status: visitQueueEntry.queueEntry.status.display,
+    waitTime: visitQueueEntry.queueEntry.startedAt
+      ? `${Math.abs(dayjs().diff(dayjs(visitQueueEntry.queueEntry.startedAt), 'minutes'))}`
```

Comment:
```suggestion
      ? `${dayjs().diff(dayjs(visitQueueEntry.queueEntry.startedAt), 'minutes')}`
```

That's `Math.abs` operating on a string. Also semantically doesn't make sense. Not from this change but let's clean it up real quick anyway.